free-2.md

```diff
@@ -1,44 +1,85 @@
 # Free project 2
 
+MarkTeX?? : Simple and possible to use in a WYSIWYG editor like Markdown,
+but targeted at LaTeX's domain.
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
 
+
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+LaTeX is way too complicated. And moreover, all the WYSIWYG attempts suck.
+What we need is a language that is as simple as possible but can still accomplish
+similar tasks.
+Moreover, we need a language with separated concerns. I don't want to use the same language 
+or process to construct my document that I use to make a new character glyph or do
+calculations.
+I don't even want to describe the specifics of styles in the same way.
```

Comment:
These are  all things that go into making a single document, just different concerns in that process. LaTeX is not just used to say "This is a problem statement," but also to say "a problem statement has a box around it" and "\cpp should be formatted by taking a C and two plus symbols, shifting those plus symbols a certain amount up an left". It is also used to do simple calculations, sometimes.  All these things are done in a pretty similar syntax.

Markdown meanwhile only does the "general layout", and the style is specified externally with CSS. That's a good thing, in my book.